src/classes/HiFiAxisConfiguration.ts

```diff
@@ -4,270 +4,63 @@
-            retval.y = inputPoint3D.y;
-        } else if (axisConfiguration.outOfScreenAxis === HiFiAxes.PositiveY && inputYIsNumber) {
-            retval.y = -inputPoint3D.y;
-        } else if (axisConfiguration.upAxis === HiFiAxes.PositiveY && inputZIsNumber) {
-            retval.y = inputPoint3D.z;
-        } else if (axisConfiguration.downAxis === HiFiAxes.PositiveY && inputZIsNumber) {
-            retval.y = -inputPoint3D.z;
-        }
-
-        if (axisConfiguration.rightAxis === HiFiAxes.PositiveZ && inputXIsNumber) {
-            retval.z = inputPoint3D.x;
-        } else if (axisConfiguration.leftAxis === HiFiAxes.PositiveZ && inputXIsNumber) {
-            retval.z = -inputPoint3D.x;
-        } else if (axisConfiguration.intoScreenAxis === HiFiAxes.PositiveZ && inputYIsNumber) {
-            retval.z = inputPoint3D.y;
-        } else if (axisConfiguration.outOfScreenAxis === HiFiAxes.PositiveZ && inputYIsNumber) {
-            retval.z = -inputPoint3D.y;
-        } else if (axisConfiguration.upAxis === HiFiAxes.PositiveZ && inputZIsNumber) {
-            retval.z = inputPoint3D.z;
-        } else if (axisConfiguration.downAxis === HiFiAxes.PositiveZ && inputZIsNumber) {
-            retval.z = -inputPoint3D.z;
-        }
-        */
-        retval = inputPoint3D;
-        return retval;
-    }
+    forward: Point3D;
 
     /**
-     * ⚠ WARNING ⚠ The code in this function IS wrong.
-     * TODO: implement the function, just a NO OP at the moment.
-     * 
-     * @param axisConfiguration 
-     * @param inputOrientationQuat3D 
+     * The 'up' direction in the World-frame.
+     * Must have length=1 and be orthogonal to 'forward'.
      */
-    static translatePoint3DFromMixerSpace(axisConfiguration: HiFiAxisConfiguration, mixerPoint3D: Point3D): Point3D {
-        let retval = new Point3D();
-        retval = mixerPoint3D;
-        return retval;
-    }
+    up: Point3D;
 
     /**
-     * ⚠ WARNING ⚠ The code in this function IS wrong.
-     * TODO: implement the function, just a NO OP at the moment.
-     * 
-     * @param axisConfiguration 
-     * @param inputOrientationQuat3D 
+     * Whether the World-frame is a right- or left-handed coordinate system.
+     *
+     * How to determine the handedness of the World coordindate system?
+     * If Z = vector_cross_product(X, Y) as per the right-hand rule
+     * (from your physics or 3D geometry class)
+     * then it is right-handed, else it is left-handed.
      */
-    static translateOrientationQuat3DToMixerSpace(axisConfiguration: HiFiAxisConfiguration, inputOrientationQuat3D: OrientationQuat3D): OrientationQuat3D {
-        let retval = new OrientationQuat3D();
-        retval = inputOrientationQuat3D;
-        return retval;
+    handedness: HiFiHandedness;
+
+    constructor(forward: Point3D, up: Point3D, handedness: HiFiHandedness) {
+        this.forward = forward;
+        this.up = up;
+        this.handedness = handedness;
```

Comment:
I find it slightly confusing/disturbing to have an enum in some places, and a left handed boolean in other places. (I have no preference between them.)